CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ $ npm i
 ## Run the app
 
 ```sh
-$ yarn dev
+$ yarn storybook
```

Comment:
Nice catch 👍 There's also the `npm run dev` command below that'll need to get updated

Reply:
✅ I'll update the `npm run dev`